File: secondmoudle/src/main/java/com/kronos/secondmoudle/TestInterceptor.java
package com.kronos.secondmoudle;

import com.kronos.router.exception.RouteNotFoundException;
import com.kronos.router.interceptor.Interceptor;

import org.jetbrains.annotations.NotNull;

public class TestInterceptor implements Interceptor {
    @Override
    public void intercept(@NotNull Chain chain) throws RouteNotFoundException {
        chain.proceed();
    }
}

File: secondmoudle/src/main/java/com/kronos/secondmoudle/TestInterceptor.kt
package com.kronos.secondmoudle

import com.kronos.router.exception.RouteNotFoundException
import com.kronos.router.interceptor.Interceptor

class TestInterceptor : Interceptor {
    @Throws(RouteNotFoundException::class)
    override fun intercept(chain: Interceptor.Chain) {
        chain.proceed()
    }
}